kolena/dataset/evaluation.py

```diff
@@ -133,7 +147,7 @@ def download_results(
                 _to_deserialized_dataframe(df_matched, column=COL_RESULT),
             ),
         )
-    log.info(f"fetched results for model '{model}' on dataset '{dataset}'")
+    log.info(f"downloaded results for model '{model}' on dataset '{dataset}'")
```

Comment:
Good catch here. Let's also update `download_dataset` to use the verb "downloaded" instead of "loaded"